foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/platform/database/Oracle18Platform.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019 IBM Corporation. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+
+// Contributors:
+//     IBM - initial drop
+//     05/06/2019 - Jody Grassel
+//       - 547023 : Add LOB Locator support for core Oracle platform.
+
+package org.eclipse.persistence.platform.database;
+
+public class Oracle18Platform extends Oracle12Platform {
+	 public Oracle18Platform() {
```

Comment:
Are you using EclipseLink's Eclipse Formatter? I notice some of the code, like this, is using tabs instead of spaces.